.eslintrc.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
     '@typescript-eslint/explicit-function-return-type': 'off',
     '@typescript-eslint/explicit-module-boundary-types': 'off',
     '@typescript-eslint/no-explicit-any': 'off',
+    "@typescript-eslint/no-unused-vars": 'off',
```

Comment:
Right below this statement is the one that turns this from a warning to an error. Either delete that or remove yours. I was asked to add that in #20

Reply:
remove this line, instead just do that is causing the error: https://github.com/codeforcauseorg/edu-server/commit/9d83abab7025c8e4496e532a8271174918b59825

Reply:
![image](https://user-images.githubusercontent.com/59202075/111968897-ac8b7600-8b1f-11eb-8996-1b042582a015.png)


Even after using UserService and CreateUserDTO no-used-vars error appeared.. that's why i wanted to remove that line @Abhishek-kumar09